javascript-practice/src/js/constants/message.js

```diff
@@ -0,0 +1,12 @@
+const VALIDATE_ERORR = {
+  Required_Field: 'This field cannot be empty!',
```

Comment:
Not the same format variable between constant file. Pls check all

Reply:
I done fix it